packages/mdx-components/src/main/anchor-links/anchor-links.stories.jsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react'
+
+import { AnchorLink, AnchorLinks } from '../../../dist/main/index'
+
+export default {
+  title: 'Anchor Links',
+  component: AnchorLinks
+}
+
+export const Normal = () => (
+  <>
+    <AnchorLinks>
+      <AnchorLink>Link 1</AnchorLink>
+      <AnchorLink>Link 2</AnchorLink>
+      <AnchorLink>Link 3</AnchorLink>
+      <AnchorLink>Link 4</AnchorLink>
+      <AnchorLink>Link 5</AnchorLink>
+      <AnchorLink>Link 6</AnchorLink>
+      <AnchorLink>Link 7</AnchorLink>
+    </AnchorLinks>
+    <div>is it red?</div>
```

Comment:
```suggestion
    <div>is it red?</div>
```
What is this for?

Reply:
Testing that was supposed to be removed 😅